files/bio.md

```diff
@@ -41,6 +48,57 @@ Jake is a Professor of Teaching in physics at UBC Okanagan.  In addition to teac
 
 ### Undergraduate students
 
+<img align="left" width="150" height="150" style="padding-right: 20px" src="../images/moon.jpeg"> </img>
+
+#### **Mrunal Mustapure**
+
+I am a second year student at UBC Okanagan. I am planning to major in Computer Science with a minor in Mathematics.
+
+I am really excited to be a part of this team and work on this project. The courses that I had taken with Dr. Moosvi were not stressful at all because of the how the courses were set with PraireLearn. To now be a part of the team working on expanding and enhancing the Open Problem Bank is a dream come true. I am really grateful to be able to have the opportunity to  enhance the experience of future students. 
+
+Aside from academia, I usually rewatch sitcoms, Modern Family, The Big Bang Theory and Brooklyn Nine-Nine are my favorites. I also like to improve and expand my speedcubing skills. I also like to collect logos. I have also recently started roaming/hiking around the campus and academy hill and it has been awesome. <br><br><br>
```

Comment:
Could you collapse these into one paragraph instead of three? Do so for this one as well as Larita's

Reply:
Update: actually I noticed a lot of the newer bios are multi-paragraph, just collapse them into one paragraph please (by remove extra line breaks!)